pyproject.toml

```diff
@@ -9,9 +9,8 @@ authors = [
 [tool.poetry.dependencies]
 python = "^3.6.2"
 rpaframework = { path = "packages/main", extras = ["cv", "playwright", "aws"] }
-rpaframework-aws = "^1.0.0"
```

Comment:
No need for `aws` here since it's included already here: `rpaframework = { path = "packages/main", extras = ["cv", "playwright", "aws"] }`, right @mikahanninen ?